public/javascripts/libraryTable.js

```diff
@@ -13,12 +13,15 @@ var libraryTableComponent = (function() {
                         m('th', 'Author'),
                         m('th', [
                             m(playlistDropdownComponent.playlistDropdown, {
-                                playlistName: playlistName
+                                playlistName: playlistName,
+                                url: '/user/library'
                             })
                         ])
                     ]),
                     m('tbody', [
                         songs().map((s) => {
+                            s.label = s.label || m.prop('Add to Playlist')
```

Comment:
I noticed that in an earlier file you have "Add to Playlist" as well. Can this default button text be passed into the component instead of hard coded?